backend/app/schemas/experiment.py

```diff
@@ -9,13 +9,15 @@
 class ExperimentBase(BaseModel):
     name: str
     description: str
-    publication_ids: list[str] = []
-
     experiment_template_id: PydanticObjectId
+
+    publication_ids: list[str] = []
     dataset_ids: list[str]
     model_ids: list[str]
+
     env_vars: list[EnvironmentVar]
     metrics: list[str]
+    is_public: bool
```

Comment:
I think it would be better to stick with simple standard terms `public` and `archived`.